src/components/ProjectCard.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+import Img from "gatsby-image";
+
+const Card = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+  padding: 20px;
+  margin: 0 0.5em;
+  margin-bottom: 1em;
+`;
+
+const ProjectDescription = styled.p`
+  text-align: center;
+  font-style: italic;
+`;
+
+const ProjectPreview = styled(Img)`
+  border: 5px solid #555;
+`;
+
+import Button from "./Button";
+
+const ProjectCard = ({ project }) => {
```

Comment:
Can you destructure the props out a little bit here so there's not loads of `project.frontmatter...` used throughout the rest of it? Would clean up the code a little bit.

Reply:
What do you actually mean by destructuring? Do you have an example?